apps/mull-ui-e2e/src/integration/US-1.1.spec.ts

```diff
@@ -87,6 +87,32 @@ frameSizes.forEach((frame) => {
       cy.get('#location ~ .error-message').should('have.text', 'Event Location is required.');
     });
 
+    it('should show an error when end time is greater than start time', () => {
```

Comment:
There seems to be a linting issue with the e2e for US-2.7, I know you didn't cause it, but if you can it would be good to fix it.

Reply:
Alright I will fix this

Reply:
For the lint error, I'm not sure if it's worth fixing, since it's complaining about the "wait(200)" for being arbitrary.  I think Ritchelle put that to make the test less flaky. What I understand is that it waits 200ms after clicking the button before the assertions, and we might need to do some stuff with `cy.route ` which might make things complicated

Reply:
no worries, was just a suggestion